File: root_VS2013_TemplateBase/programs/VB/Samples/2CS_sample/GenDaoAndBatUpd_sample/Common/TestParameterValue.vb
'**********************************************************************************
'* フレームワーク・テストクラス
'**********************************************************************************

'**********************************************************************************
'* クラス名        ：TestParameterValue
'* クラス日本語名  ：テスト用の引数クラス
'*
'* 作成日時        ：－
'* 作成者          ：sas 生技
'* 更新履歴        ：
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'**********************************************************************************

Imports System
Imports System.Data

' ベースクラス
Imports Touryo.Infrastructure.Business.Util
Imports Touryo.Infrastructure.Business.Common

Namespace Common
    ''' <summary>
    ''' TestParameterValue の概要の説明です
    ''' </summary>
    Public Class TestParameterValue
        Inherits MyParameterValue
#Region "コンストラクタ"

        Public Sub New(ByVal screenId As String, ByVal controlId As String, ByVal methodName As String, ByVal actionType As String, ByVal user As MyUserInfo)
            ' Baseのコンストラクタに引数を渡すために必要。
            MyBase.New(screenId, controlId, methodName, actionType, user)
        End Sub

#End Region

#Region "フィールド"

        ' 値（インサート、主キー値など）
        Public field1 As Object
        Public field2 As Object
        Public field3 As Object
        Public field4 As Object
        Public field5 As Object
        Public field6 As Object
        Public field7 As Object
        Public field8 As Object
        Public field9 As Object
        Public field10 As Object
        Public field11 As Object
        Public field12 As Object
        Public field13 As Object
        Public field14 As Object
        Public field15 As Object
        Public field16 As Object
        Public field17 As Object
        Public field18 As Object
        Public field19 As Object
        Public field20 As Object

        ' 更新時
        Public field1_ForUpd As Object
        Public field2_ForUpd As Object
        Public field3_ForUpd As Object
        Public field4_ForUpd As Object
        Public field5_ForUpd As Object
        Public field6_ForUpd As Object
        Public field7_ForUpd As Object
        Public field8_ForUpd As Object
        Public field9_ForUpd As Object
        Public field10_ForUpd As Object
        Public field11_ForUpd As Object
        Public field12_ForUpd As Object
        Public field13_ForUpd As Object
        Public field14_ForUpd As Object
        Public field15_ForUpd As Object
        Public field16_ForUpd As Object
        Public field17_ForUpd As Object
        Public field18_ForUpd As Object
        Public field19_ForUpd As Object
        Public field20_ForUpd As Object

        ' 検索条件
        Public field1_ForSearch As Object
        Public field2_ForSearch As Object
        Public field3_ForSearch As Object
        Public field4_ForSearch As Object
        Public field5_ForSearch As Object
        Public field6_ForSearch As Object
        Public field7_ForSearch As Object
        Public field8_ForSearch As Object
        Public field9_ForSearch As Object
        Public field10_ForSearch As Object
        Public field11_ForSearch As Object
        Public field12_ForSearch As Object
        Public field13_ForSearch As Object
        Public field14_ForSearch As Object
        Public field15_ForSearch As Object
        Public field16_ForSearch As Object
        Public field17_ForSearch As Object
        Public field18_ForSearch As Object
        Public field19_ForSearch As Object
        Public field20_ForSearch As Object

#End Region

        Public dt As DataTable
        Public obj As Object
    End Class
End Namespace

File: root/programs/CS/Samples/2CS_sample/GenDaoAndBatUpd_sample/Common/TestParameterValue.cs
//**********************************************************************************
//* バッチ更新処理・サンプル アプリ
//**********************************************************************************

// テスト用サンプルなので、必要に応じて流用 or 削除して下さい。

//**********************************************************************************
//* クラス名        ：TestParameterValue
//* クラス日本語名  ：テスト用の引数クラス
//*
//* 作成日時        ：－
//* 作成者          ：生技
//* 更新履歴        ：
//*
//*  日時        更新者            内容
//*  ----------  ----------------  -------------------------------------------------
//*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
//**********************************************************************************

using System.Data;
using Touryo.Infrastructure.Business.Common;
using Touryo.Infrastructure.Business.Util;

namespace GenDaoAndBatUpd_sample.Common
{
    /// <summary>
    /// TestParameterValue の概要の説明です
    /// </summary>
    public class TestParameterValue : MyParameterValue
    {
        #region コンストラクタ

        public TestParameterValue(string screenId, string controlId, string methodName, string actionType, MyUserInfo user)
            : base(screenId, controlId, methodName, actionType, user)
        {
            // Baseのコンストラクタに引数を渡すために必要。
        }

        #endregion

        #region フィールド

        // 値（インサート、主キー値など）
        public object field1;
        public object field2;
        public object field3;
        public object field4;
        public object field5;
        public object field6;
        public object field7;
        public object field8;
        public object field9;
        public object field10;
        public object field11;
        public object field12;
        public object field13;
        public object field14;
        public object field15;
        public object field16;
        public object field17;
        public object field18;
        public object field19;
        public object field20;

        // 更新時
        public object field1_ForUpd;
        public object field2_ForUpd;
        public object field3_ForUpd;
        public object field4_ForUpd;
        public object field5_ForUpd;
        public object field6_ForUpd;
        public object field7_ForUpd;
        public object field8_ForUpd;
        public object field9_ForUpd;
        public object field10_ForUpd;
        public object field11_ForUpd;
        public object field12_ForUpd;
        public object field13_ForUpd;
        public object field14_ForUpd;
        public object field15_ForUpd;
        public object field16_ForUpd;
        public object field17_ForUpd;
        public object field18_ForUpd;
        public object field19_ForUpd;
        public object field20_ForUpd;

        // 検索条件
        public object field1_ForSearch;
        public object field2_ForSearch;
        public object field3_ForSearch;
        public object field4_ForSearch;
        public object field5_ForSearch;
        public object field6_ForSearch;
        public object field7_ForSearch;
        public object field8_ForSearch;
        public object field9_ForSearch;
        public object field10_ForSearch;
        public object field11_ForSearch;
        public object field12_ForSearch;
        public object field13_ForSearch;
        public object field14_ForSearch;
        public object field15_ForSearch;
        public object field16_ForSearch;
        public object field17_ForSearch;
        public object field18_ForSearch;
        public object field19_ForSearch;
        public object field20_ForSearch;

        #endregion

        public DataTable dt;
        public object obj;
    }
}
